bc_obps/service/transfer_event_service.py

```diff
@@ -0,0 +1,32 @@
+from registration.models.event.transfer_event import TransferEvent
+from typing import Optional, Dict, Any, List
+
+
+from registration.schema.v1.transfer_event import TransferEventFilterSchema
+from ninja import Query
+
+
+class TransferEventService:
+    @classmethod
+    def list_transfer_events(
+        cls,
+        sort_field: Optional[str],
+        sort_order: Optional[str],
+        filters: TransferEventFilterSchema = Query(...),
+    ) -> List[Dict[str, Any]]:
```

Comment:
mypy said that you can't have a QuerySet[List], it has to be QuerySet[ModelName], so I've gone with this